jiratools/formatting.py

```diff
@@ -1,13 +1,18 @@
 """Formatting helpers for jiratools."""
 import getpass
 import os
+from typing import Optional, List, Union
 
+DataArray = List[List[Union[str, int]]]
```

Comment:
Man I don't like how you have to explicitly say `List` like this. Maybe it would be cleaner if you split out the contents of the list.

Also, I had to go read `data_array: An array of lists, representing table rows.` to know what this is meant to be. Perhaps the naming of the typing here can help with that.

```py
TableRowData =  List[Union[str, int]]
DataArray = List[TableRowData]
```

I might even suggest changing `DataArray` to something like `TableRows`, but that has larger implications than this PR. Related side note: Kind of odd that we use list and array back and forth. Certainly they are the same, but might be worth unifying at some point.